recipe/meta.yaml

```
@@ -12,7 +12,7 @@ source:
   sha256: 825919da8957bbc19cec35caf8663b734d34af72a0b040c43b7d8c1b76fdcab7
 
 build:
-  number: 0
+  number: 1
```

```suggestion
  number: 0
```
As 3.10.0 was not uploaded (from what I can see).